src/Receipt/DonorDetailsGroup/Details/Email.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Give\Receipt\DonorDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_donation_donor_email as getDonationDonorEmail;
+
+/**
+ * Class Email
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonorDetailsGroup\Details
+ */
+class Email extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'EMAIL ADDRESS', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.